hack/update-generated-crd-code.sh

```diff
@@ -42,7 +42,7 @@ ${GOPATH}/src/k8s.io/code-generator/generate-groups.sh \
   $@
 
 go run ${GOPATH}/src/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go \
-  crd:crdVersions=v1beta1,preserveUnknownFields=false \
+  crd:'crdVersions={v1,v1beta1}' \
```

Comment:
what would be the version of the CRDs stored?
And do we have to perform any conversions between versions?

Reply:
Controller-tools generates both v1 and v1beta1 manifest.
When installing, Velero server pick the server preferred API vesion to install.